src/main/java/com/gtnewhorizons/gtnhgradle/modules/IdeIntegrationModule.java

```diff
@@ -177,9 +177,12 @@ public void apply(GTNHGradlePlugin.@NotNull GTNHExtension gtnh, @NotNull Project
             final RunMinecraftTask runClient = tasks.named("runClient", RunMinecraftTask.class)
                 .get();
             final var run = ijClientRun.get();
-            run.setWorkingDirectory(
-                runClient.getWorkingDir()
-                    .getAbsolutePath());
+            String runDir = gtnh.configuration.separateRunDirs ? runClient.getWorkingDir()
```

Comment:
Please use the File() constructor or FileUtils path joining functions from apachecommons instead of manually concatenating the separator string